### PR TITLE
Add initial metric

### PR DESCRIPTION
**Pull request by gsvr**

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,11 @@
...
 Seed Message Sender and Message Store
+
+## Apps & Models:
+  * message_sender
+    * Outbound
+    * Inbound
+
+## Metrics
+##### inbounds.created.sum
+`sum` Total number of inbounds created
```
Comment (imsickofmaps) [inline README.md:11]: missing some here

File: seed_message_sender/settings.py [hunk 2]
```diff
@@ -171,14 +171,21 @@
...
     },
-    'message_sender.tasks.send_metric': {
-        'queue': 'lowpriority',
+    'message_sender.tasks.fire_metric': {
+        'queue': 'metrics',
     },
 }
 
+METRICS_REALTIME = [
+    'subscriptions.created.sum'
```
Comment (imsickofmaps) [inline seed_message_sender/settings.py]: missing some here

**Comment by imsickofmaps** [comment]
Minor docs issues then 👍

**Pull request merged**